style.css

```diff
@@ -41,3 +41,180 @@
 .green {
   color: #36b37e;
 }
+
+.line {
+  width: 48px;
+  height: 4px;
+  background-color: #36b37e;
+}
+
+.work-section {
+  width: 100%;
+  align-items: cente;
+  padding: 10px 15px 15px 10px;
+}
+
+.project-container {
+  display: grid;
+  grid-template-columns: 80%;
+  grid-template-rows: auto auto auto auto auto auto;
+  gap: 20px;
+}
+
+.project {
+  border: 1px solid green;
+  margin: 0;
+  width: 100%;
+  padding: 0;
```

Comment:
![Screenshot from 2022-09-23 13-30-44](https://user-images.githubusercontent.com/65233860/191971880-d74825fc-59d8-4832-a87a-a549e4cab03a.png)


As shown in the image above, your project card has shifted to the left side, Kindly check the margins and other dimensions of the figma design so that it will be centred just as it's suppose to be.

Figma design

![Screenshot from 2022-09-23 13-32-11](https://user-images.githubusercontent.com/65233860/191972384-451f2520-3ef8-400b-84f5-53521aa96eaf.png)